cpp/include/cudf/unary.hpp

```diff
@@ -65,41 +66,47 @@ enum class unary_operator : int32_t {
  *
  * @param input A `column_view` as input
  * @param op operation to perform
+ * @param stream CUDA stream used for device memory operations and kernel launches.
```

Comment:
The period is not removed after copying :)
(not just here but all around this file)

Reply:
Was that something that we wanted to do? I can do it, I didn't remember.